src/c_regex.zig

```diff
@@ -4,28 +4,72 @@ const std = @import("std");
 
 const regex = @import("regex.zig");
 const Regex = regex.Regex;
+const Captures = regex.Captures;
 
-const zre_regex = opaque {};
+const zre_regex_t = opaque {};
```

Comment:
Can you rename these without to omit the `_t` suffix.

Reply:
I think it's more convenient to use `_t` for types.

Reply:
also if I want to remove it I have to do it for other types and it will result of conflicted names like `zre_captures`

Reply:
I excluded `_t` initially because these are reserved as type names in POSIX. Not that I expect conflicts here but would prefer to keep the original for this sake.